src/MIDebugEngine/Engine.Impl/Variables.cs

```diff
@@ -365,7 +365,7 @@ public enum NodeType
 
         private static Regex s_isFunction = new Regex(@".+\(.*\).*");
 
-        private string StripFormatSpecifier(string exp, out string formatSpecifier)
+        private string StripFormatSpecifier(string exp, out string formatSpecifier, int threadId, uint frameLevel)
```

Comment:
>StripFormatSpecifier [](http://example.com/codeflow?start=23&length=20)

nit: Maybe rename to `ProcessFormatSpecifiers` since this now does a lot more than 'stripping'